src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.data.person;
+
+
+import seedu.addressbook.data.tag.Tag;
+
```

Comment:
Please include a header comment to indicate what this interface/class/method does?